src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,212 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare the sizes of shards containing
+varying amounts of ledgers.
+
+## Methodology
+
+One can see visually from a block explorer that typical mainnet ledger
+consists of about 30 offer transactions issued by about 8 different accounts,
+and several transactions of other types. To simulate this situation and
+similar situations we had constructed deterministic shards of differenet sizes,
+with different number of offer transactions per ledger and different number
+of accounts issuing these offers.
+
+Number of ledgers per shard starts from 256 to 16K with doubling the size on
```

Comment:
Consider: "In the following results table, the number of ledgers per shard ranges from 256 to 16K with the size doubling the size at each step"

Reply:
Done.